test/table-format.vader

```diff
@@ -30,6 +30,19 @@ Expect (table is not modified):
   |---|---|
   | c | d |
 
+Given markdown (indented table):
+    | a | b |
+    |---|---|
+    | c | d |
+
+Execute (format well formatted, indented table):
+  TableFormat
+
+Expect (table is not modified):
+    | a | b |
```

Comment:
With the `Given` input having 4 space indent, I think the `Expect` output needs to have 6.

Reply:
The old tests use `;` at the end of the block label instead of `:`. If `:` is used, the content should be indented with 2 spaces. If `;` is used, the indentation is unnecessary. I think it's very confusing to mix `;` and `:`, but anyway in the latest force push I changed `:` in the expect label to `;` to follow the style of other test cases.

Reply:
Aaa, okay thanks for the clarification. I wasn't sure what was up there. I agree it's confusing to mix, I'll be happy to sort them all out into the same style eventually.